src/screens/ToggleGroup.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { ToggleGroup } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+import { GenericPad, GenericBoolFalse } from '../utils/genericPropExamples';
+
+const ToggleGroupPage = () => (
+  <Page>
+    <ComponentDoc
+      name=""
+      availableAt={[
+        {
+          url: '',
+          badge:
+            'https://storybook.grommet.io/?path=/story/controls-togglegroup-simple--simple',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://github.com/grommet/grommet/tree/master/src/js/components/ToggleGroup',
+          label: 'Github',
+        },
+      ]}
+      description="A collection of binary buttons capable of being switched between two states: on or off."
```

Comment:
```suggestion
      description="A collection of toggle buttons capable of being switched between two states: on or off."
```
Is "toggle" a more familiar term than binary maybe?